### PR TITLE
[rabbitmq] provide option to set service externalTrafficPolicy and type

### PR DESCRIPTION
**Pull request by defo89**

### CODE REVIEW

**Review by bashar-alkhateeb — COMMENTED** [review]
would you kindly add a section for this change to the CHANGELOG.md other wise looks good

File: common/rabbitmq/templates/service.yaml [hunk 1]
```diff
@@ -21,6 +21,10 @@ metadata:
     linkerd.io/inject: enabled
 {{- end }}
 spec:
+  type: {{ default "ClusterIP" .Values.service.type }}
```
Comment (s10) [inline common/rabbitmq/templates/service.yaml:24]: I get this error when generating template with test values
```
Error: template: rabbitmq/templates/service.yaml:24:38: executing "rabbitmq/templates/service.yaml" at <.Values.service.type>: nil pointer evaluating interface {}.type
helm.go:84: [debug] template: rabbitmq/templates/service.yaml:24:38: executing "rabbitmq/templates/service.yaml" at <.Values.service.type>: nil pointer evaluating interface {}.type
```
The same error in ci/concourse/test-helm-chart

Comment (fwiesel) [reply common/rabbitmq/templates/service.yaml:24]: I'd move the default straight to the values. I.e.
Here simply: `{{ .Values.service.type }}`
And in the values
```
service:
  type: "ClusterIP"
```
That fixes the problem and personally, I'd prefer my defaults to be in the values instead being spread over the templates.

Comment (defo89) [reply common/rabbitmq/templates/service.yaml]: hope the last fix addresses all issues

**Review by s10 — APPROVED** [review]
lgtm

File: common/rabbitmq/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: rabbitmq
-version: 0.9.0
+version: 0.9.1
```
Comment (businessbean) [inline common/rabbitmq/Chart.yaml:3]: Because of the added feature it has to be a `0.10.0`.

**Review by businessbean — CHANGES_REQUESTED** [review]
lgtm. Just change the version...

File: common/rabbitmq/Chart.yaml (hunk 2 above)
Comment (defo89) [reply common/rabbitmq/Chart.yaml]: sure, changed the version

**Review by businessbean — APPROVED** [review]
lgtm

**Pull request merged**